### PR TITLE
First attempt to validate our XSLT stylesheets

### PR DESCRIPTION
**Pull request by tomschr**
* Using `xslt10.rnc` RELAX NG schema
* Validate with jing
* Iterate over all `suse2013` subdirectories for the time being

### CODE REVIEW

File: .travis/xslt10.rnc [hunk 1]
```diff
@@ -0,0 +1,356 @@
+# $Id: xslt10.rnc 52 2006-07-12 14:24:40Z ndw $
```
Comment (sknorr) [inline .travis/xslt10.rnc]: Hm, might it make sense to use a git submodule for this? If not, maybe add some information about the origin/license of this file...

Comment (tomschr) [reply .travis/xslt10.rnc]: Yes, in the first place I had this checked out directly from the GitHub repository. I don't think it makes much sense to use a submodule here as a) the RNG schema doesn't change anymore (hopefully) and b) checking out a whole GH repo just for a single file is a bit pointless (yes, you could download the raw file without checking out all).

Unfortunately, it seems, the GH repo doesn't have license information...

File: .travis/validate-xslt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+RNG="$MYPATH/xslt10.rnc"
+
+echo $RNG
+if [[ ! -e $RNG ]]; then
+ echo "Couldn't find the RNG schema"
+ exit 10
+fi
+
+# These are the directories to investigate:
+DIRS=$(echo "suse2013/common/*.xsl suse2013/epub3/*.xsl suse2013/fo/*.xsl suse2013/xhtml/*.xsl suse2013/xhtml5/*.xsl")
```
[the remark this answers is not in the document]
Comment (tomschr) [reply .travis/validate-xslt]: Right, I've added two "config" files now which can also have comment lines starting with hash characters.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ precise main universe restricted multiverse"
+    - sudo apt-get update
+    - sudo apt-get install -y jing
+
+install:
+    - echo "Installation complete"
+
+script:
+    - ./.travis/validate-xslt
```
[the remark this answers is not in the document]
Comment (tomschr) [reply .travis.yml]: Ok, done.

**Comment by tomschr** [comment]
I took the liberty to merge it to develop now so we can use this. Thanks for the feedback! 👍

**Pull request closed without merging**